akka-persistence-query/src/main/scala/akka/persistence/query/typed/EventEnvelope.scala

```diff
@@ -58,7 +81,22 @@ final class EventEnvelope[Event](
     val timestamp: Long,
     val eventMetadata: Option[Any],
     val entityType: String,
-    val slice: Int) {
+    val slice: Int,
+    val filtered: Boolean,
+  val source: String) {
+
+  // backwards compatibility when adding filtered and source
+  @deprecated("Use constructor with all parameters", "2.8.0-M4")
+  def this(
```

Comment:
Do we need to deprecate this, I think for journals not doing filtering and backtracking this constructor is fine?

Reply:
We have `apply` and `create` factory methods and there I kept the old signatures. I agree, no need to deprecate this constructor.